src/xpcc/architecture/platform/driver/adc/stm32f0/adc.hpp.in

```diff
@@ -0,0 +1,263 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2018, Álan Crístoffer
+ *
+ * This file is part of the xpcc project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
Sorry, I meant that all code in xpcc is BSD licensed, but every quarter I'm forward merging all changes in xpcc over to modm and changing the license to MPLv2. So effectively you're dual-licensing your code as BSD in xpcc and MPLv2 in modm. This is a practical compromise to allow continued xpcc development, while we radically refactor this code in modm.

Reply:
No problem, I agree.

Reply:
Please change this back to the BSD style header we use in xpcc.